services/orders_create/lambda_function.py

```diff
@@ -0,0 +1,54 @@
+import os
+import json
+import uuid
+import boto3
+import decimal
+from botocore.exceptions import ClientError
+
+# get the service resource
+dynamodb = boto3.resource('dynamodb')
+# get the users table
+ORDERS_TABLE = os.environ['ORDERS_TABLE']
+
+# helper class to convert a DynamoDB item to JSON
+# for details see: https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/GettingStarted.Python.03.html
+class DecimalEncoder(json.JSONEncoder):
+    def default(self, o):
+        if isinstance(o, decimal.Decimal):
+            if abs(o) % 1 > 0:
+                return float(o)
```

Comment:
Conversion to float might bite us in the ass at a later point. At the same time, if the teachers expect us to use JSON numbers, then we have no choice.